package.json

```diff
@@ -25,6 +25,7 @@
     "antd": "3.6.2",
     "codemirror": "5.38.0",
     "color": "3.0.0",
+    "file-saver": "1.3.8",
```

Comment:
Do we really need a lib for that? Not sure, just asking.

Reply:
The library covers multiple browsers. For modern Browsers we wouldn't need it.